### PR TITLE
Ignore JetBrains classes during instrumentation

### PR DESCRIPTION
**Pull request by bertschneider**
Some JetBrains products interact badly with Jazzer's agent / instrumentation, e.g. their coverage agent. It's reasonable to ignore all JetBrains classes.

### CODE REVIEW

File: src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt [hunk 1]
```diff
@@ -181,7 +181,10 @@ class RuntimeInstrumentor(
             .acceptClasses(className)
             .scan()
             .use {
-                it.getClassInfo(className).resource.load()
+                it.getClassInfo(className)?.resource?.load() ?: run {
```
Comment (fmeum) [inline src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt:184]: Could you mention this change in the commit message or split it out into a separate commit?

**Review by fmeum — APPROVED** [review]

File: src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt (hunk 1 above)
Comment (bertschneider) [reply src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt:184]: done.
